File: GotlandBackend/Models/BookingPaxListItem.cs
using System;

namespace Accidis.Gotland.WebService.Models
{
	public class BookingPaxListItem
	{
		public Guid Id { get; set; }
		public string Reference { get; set; }
		public string TeamName { get; set; }
		public string FirstName { get; set; }
		public string LastName { get; set; }
		public string Gender { get; set; }
		public string Dob { get; set; }
		public int CabinClassMin { get; set; }
		public int CabinClassPreferred { get; set; }
		public int CabinClassMax { get; set; }
	}
}

File: GotlandFrontend/lib/model/booking_pax_list_item.dart
import 'json_field.dart';

class BookingPaxListItem {
	final String reference;
	final String teamName;
	final String firstName;
	final String lastName;
	final String gender;
	final String dob;
	final int cabinClassMin;
	final int cabinClassPreferred;
	final int cabinClassMax;

	BookingPaxListItem(this.reference, this.teamName, this.firstName, this.lastName, this.gender, this.dob, this.cabinClassMin, this.cabinClassPreferred, this.cabinClassMax);

	factory BookingPaxListItem.fromMap(Map<String, dynamic> json) =>
		BookingPaxListItem(
			json[REFERENCE],
			json[TEAM_NAME],
			json[FIRSTNAME],
			json[LASTNAME],
			json[GENDER],
			json[DOB],
			json[CABIN_CLASS_MIN],
			json[CABIN_CLASS_PREF],
			json[CABIN_CLASS_MAX]
		);
}
